bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/binding/builder/ChannelBuilder.java

```diff
@@ -105,6 +105,10 @@ public static ChannelBuilder create(Channel channel) {
         if (description != null) {
             channelBuilder.withDescription(description);
         }
+        AutoUpdatePolicy autoUpdatePolicy = channel.getAutoUpdatePolicy();
```

Comment:
Since `.withAutoUpdatePolicy` accepts `null` you can just use that in l. 99

Reply:
Right - done.